### ISSUE TITLE
Make problem

### ISSUE THREAD

**Issue by ras2017it6** [hartfordfive/protologbeat#6]
after running make we get : 
![capture789789789](https://user-images.githubusercontent.com/29635181/27917538-30627ffc-6264-11e7-8151-e14ee94d635a.PNG)

**Comment by hartfordfive** [comment]
@ras2017it6 Someone also mentioned an issue with the make script in issue #5.   I'll look into this as soon as I get a chance. Until then, you can get a pre-compiled build for your system by just downloading it from the [releases](https://github.com/hartfordfive/protologbeat/releases) section.

**Comment by raabdo** [comment]
Okay thank you, but I want to make some changes in the config file to use Sys Log protocol is it possible with the pre-compiled version
And I want to ask you about UDPlogbeat, is it compiling without issues ??

**Comment by hartfordfive** [comment]
Yes, you can enable syslog-only mode via the config.  Here's a a [simple example configuration](https://github.com/hartfordfive/protologbeat/blob/master/_samples/config3.yml).  As for [udplogbeat](https://github.com/hartfordfive/udplogbeat), it also has pre-compiled binaries for it although I'm no longer maintaining/updating it as protologbeat is the successor.

**Comment by ras2017it6** [comment]
to break the cycle we are supposed to have the conception of the project, could you sent us the uml diagram of the whole project to implement the interface correctly? is there a precompiled file with a enableonlysyslogformat=true set by default ?
